### PR TITLE
Update eds-icons with latest changes

### LINKED ISSUE #588: Update eds-icons with latest changes
#resolves #545 
blocked by #545

### PR DESCRIPTION
**Pull request by mimarz**
* new `filter_alt` icon
* updates to `info_circle` & `list` icon

resolves #588

### CODE REVIEW

File: libraries/static/icons/system-icons/ui-views/filter_alt.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M7 6h10l-5.01 6.3L7 6zm-2.75-.39C6.27 8.2 10 13 10 13v6c0 .55.45 1 1 1h2c.55 0 1-.45 1-1v-6s3.72-4.8 5.74-7.39A.998.998 0 0018.95 4H5.04c-.83 0-1.3.95-.79 1.61z"/></svg>
```
Comment (vnys) [inline libraries/static/icons/system-icons/ui-views/filter_alt.svg:1]: Missing new line at end of file

File: libraries/static/icons/system-icons.svg [hunk 2]
```diff
@@ -2516,6 +2511,5 @@
       <desc>accessibility-translate</desc>
       <path fill-rule="evenodd" clip-rule="evenodd" d="M12.87 15.07l-2.54-2.51.03-.03A17.52 17.52 0 0014.07 6H17V4h-7V2H8v2H1v1.99h11.17C11.5 7.92 10.44 9.75 9 11.35 8.07 10.32 7.3 9.19 6.69 8h-2c.73 1.63 1.73 3.17 2.98 4.56l-5.09 5.02L4 19l5-5 3.11 3.11.76-2.04zM18.5 10h-2L12 22h2l1.12-3h4.75L21 22h2l-4.5-12zm-1 2.67L15.88 17h3.24l-1.62-4.33z"/>
     </symbol>
-      </defs>
     </svg>
```
Comment (vnys) [inline libraries/static/icons/system-icons.svg:2515]: Missing new line at end of file

**Review by vnys — APPROVED** [review]
Missing newlines at the end of two of the files. After that it LGTM 👍

**Comment by mimarz** [comment]
@vnys We have no new lines at end of file on any of our svg og json files from figma-broker. Fixing this results in over 630 file changes, so saving it for a seperate fix here #599 😅

File: libraries/static/icons/system-icons.svg (hunk 2 above)
Comment (mimarz) [reply libraries/static/icons/system-icons.svg:2515]: will be fixed in #599

File: libraries/static/icons/system-icons/ui-views/filter_alt.svg (hunk 1 above)
Comment (mimarz) [reply libraries/static/icons/system-icons/ui-views/filter_alt.svg:1]: will be fixed in #599

**Pull request merged**